brave/browser-laptop
# Refactor contributionStatement.js with Aphrodite

Closes #7883

- Remove contributionStatement.less
- Replace inlined braveLogo with a PNG file, verifiedIcon with a SVG file
- Introduce constants for margins, borders, paddings, light gray, etc

Auditors:


Test Plan:
1. npm run add-simulated-payment-history
2. Open about:preferences#payments
3. Display the payments history
4. Save the PDF file
5. Open it and make sure it is successfully rendered
